/**:
  ros__parameters:
    cmd_freq: 30.0  # Hz (default: 100.0)
    info_freq: 10.0  # Hz (default: 10.0)
    enable_camera: false  # (default: false)
    enable_gimbal: false  # (default: false)
    gimbal_base_frame_id: "gimbal_base"  # (default: "gimbal_base")
    tf_timeout_threshold: 0.05 # Default tf timeout (50ms)